package.json

```diff
@@ -83,7 +83,18 @@
                             "type": "string",
                             "default": null,
                             "description": "File path to a draft binary."
+                        },
+                        "vs-kubernetes.autoCleanupOnDebugTerminate": {
+                            "type": "boolean",
+                            "default": true,
+                            "description": "Once the debug session is terminated, automatically clean up the created Deployment and associated Pod by the command \"Kubernetes: Debug (Launch)\"."
                         }
+                    },
+                    "default": {
```

Comment:
What does this do?  How do we keep it in sync with the `default` value in each contributed config entry?

Reply:
> What does this do?

Because we wrap a parent level for `vs-kubernetes.xxx`, and vscode doesn't show the details in `DEFAULT SETTINGS` view. Adding an explicit `default` field for the parent level `vs-kubernetes` will help list each config items in `DEFAULT SETTINGS` view. See the snapshot below.
![image](https://user-images.githubusercontent.com/14052197/37754507-cbf0d466-2ddc-11e8-92a5-77c2497cea41.png)

> How do we keep it in sync with the default value in each contributed config entry?

I did some experiment for this. The `default` value for the second level settings `vs-kubernetes.xxx` doesn't take any effect if i leave user setting to be empty, they all are `undefined` from the VS Code API `vscode.workspace.getConfiguration("vs-kubernetes")`. It seems only the first level `default` value has impact and shows up in `DEFAULT SETTINGS` view.

I'm going to remove the `default` value definition from each config entry, and put them in the parent level together.